frontend/app-development/features/administration/components/Administration.module.css

```diff
@@ -1,35 +1,77 @@
-.avatar {
-  max-height: '2em';
+.administration {
+  background-color: #e6eff8;
+  position: relative;
 }
-.sidebarHeader {
-  margin-bottom: 20;
-  font-size: 20;
-  font-weight: 500;
+
+.administration::before {
+  /* webpackIgnore: true */
+  background-image: url('/designer/img/Altinn-studio.svg');
+  background-position: center -220px;
+  background-size: 1620px auto;
+  background-repeat: no-repeat;
+
+  content: ' ';
+  display: block;
+  position: absolute;
+  margin: auto;
+  width: 100%;
+  height: 100%;
+  opacity: 0.1;
+}
+
+.container {
+  margin: auto;
+  padding: var(--fds-spacing-12);
+  max-width: 1400px;
+  position: relative;
+}
+
+.header {
+  font-family: 'Inter', sans-serif;
```

Comment:
Here we can use the new --studio-font-family variable. 😊

Reply:
Nice catch, done!